### PR TITLE
Please merge fix

### PR DESCRIPTION
**Pull request by ascheman**

### CODE REVIEW

File: pkg/jenkinsfile/pipeline.go [hunk 1]
```diff
@@ -571,9 +571,11 @@ func (a *CreateJenkinsfileArguments) GenerateJenkinsfile(resolver ImportFileReso
 	}
 	outFile := a.OutputFile
 	outDir, _ := filepath.Split(outFile)
-	err = os.MkdirAll(outDir, util.DefaultWritePermissions)
-	if err != nil {
-		return errors.Wrapf(err, "failed to make directory %s", outDir)
+	if outDir != "" {
+		err = os.MkdirAll(outDir, util.DefaultWritePermissions)
+		if err != nil {
+			return errors.Wrapf(err, "failed to make directory '%s' when creating Jenkinsfile '%s'", outDir, outFile)
```
Comment (pmuir) [inline pkg/jenkinsfile/pipeline.go]: Can you remove the single quotes around the variables to be consistent with the rest of Jenkins X please

**Comment by ascheman** [comment]
Where can I see the test result of the failed test, @pmuir ? Or how could I even test this locally? Which test case failed in particular?

**Comment by pmuir** [comment]
Ah same BDD tests issues

**Comment by ascheman** [comment]
Will ignore these failures, hoping you get it fixed soon.

**Pull request merged**